packages/rules/README.md

```diff
@@ -0,0 +1,55 @@
+# @sharegate/eslint-plugin-rules
+
+Sharegate&#39;s custom ESLint plugin.
+
+## Installation
+
+You'll first need to install [ESLint](http://eslint.org):
+
+```
+$ npm i eslint --save-dev
+```
+
+Next, install `@sharegate/eslint-plugin-rules`:
+
+```
+$ npm install @sharegate/eslint-plugin-rules --save-dev
+```
+
+**Note:** If you installed ESLint globally (using the `-g` flag) then you must also install `@sharegate/eslint-plugin-rules` globally.
+
+## Usage
+
+Add `sharegate` to the plugins section of your `.eslintrc` configuration file. You can omit the `eslint-plugin-` prefix:
```

Comment:
Je dirais simplement

Add the sharegate rules plugin to your `.eslintrc` configuration file:
```javascript
{
    "plugins": ["@sharegate/rules"]
}
```